contents/tutorials/react-analytics.md

```diff
@@ -104,7 +104,7 @@ First install `posthog-js`:
 npm install posthog-js
 ```
 
-Next, import PostHog into `src/main.jsx` and set up it up using your project API key and host from [your project settings](https://us.posthog.com/settings/project). Then we wrap our app with `PostHogProvider` to access PostHog in any component.
+Next, import PostHog into `src/main.jsx` and set up it up using your project API key and host from [your project settings](https://us.posthog.com/settings/project). Then we wrap our app with `PostHogProvider` to access PostHog in any component. We set `capture_pageview` to `history_change` because React Router acts as a single-page app and doesn't fire page load events on route changes.
```

Comment:
```suggestion
Next, import PostHog into `src/main.jsx` and set up it up using your project API key and host from [your project settings](https://us.posthog.com/settings/project). Then we wrap our app with `PostHogProvider` to access PostHog in any component. We set `capture_pageview` to `history_change` because a single page application using React Router doesn't fire page load events on route changes.
```

I think in the case of react + react router, it literally is an SPA (?) correct me if wrong.

Reply:
I'm not really sure what the best way to work this is, but I'll just stick with mine